fecgov/openFEC
# Update check_amount to payment_amount

## Summary (required)

- Resolves #4853 

Update parameter name `check_amount` to `payment_amount`.

### Required reviewers

1 backend developer

## Impacted areas of the application

General components of the application that this PR will affect:

-  `/legal/docs/admin_fines/` endpoint

## Screenshots

![Screen Shot 2021-05-25 at 1 20 16 PM](https://user-images.githubusercontent.com/12799132/119540960-07c62880-bd5c-11eb-8be6-a5ce4a0d3d77.png)

## How to test

- Checkout this branch and then branch off this branch to deploy to the dev API
- Tail the API dev logs: `cf target -s dev && cf logs api`
- In a separate terminal window, run a task to load a single AF case like this: `cf run-task api --command "python manage.py load_admin_fines -s 26" --name load_admin_fines`. Confirm in the logs that the single AF case has been loaded
- Go to the dev API endpoint and check that `check_amount` is now `payment_amount`. 

## Follow-up task
After this is merged in, we need to recreate the ElasticSearch index and mapping so that it's synchronized. Documentation to follow is here: https://github.com/fecgov/openFEC/wiki/Elasticsearch-7.x.0-management-instruction#1-index-management.
